src/login/login.component.tsx

```diff
@@ -210,6 +210,9 @@ const Login: React.FC<LoginReferrer> = () => {
               <a href="mailto:emrtalk@musph.ac.ug">emrtalk@musph.ac.ug</a> for
               support
             </span>
+            <span className={styles.link}>
```

Comment:
highlight this to indicate that its a link otherwise seems like any nomal text . You could use something link `<a href="#"></a>`